app/views/static_pages/home.html.erb

```diff
@@ -10,3 +10,22 @@
   <br>
   <%= Date.today %>
 </p>
+
+<% if logged_in? %>
+  <div class="row">
+    <aside class="col-md-4">
+      <section class="user_info">
+        <%= render "shared/user_info" %>
+      </section>
+      <section class="micropost_form">
+        <%= render "shared/micropost_form" %>
+      </section>
+    </aside>
+    <div class="col-md-8">
+      <h3>Micropost Feed</h3>
+      <%= render "shared/feed" %>
+    </div>
+  </div>
+<% else %>
+...
```

Comment:
you seem missing some code handler in here
please read and follow tutorial carefully